tests/func/test_datachain.py

```diff
@@ -463,3 +465,162 @@ def test_mutate_existing_column(test_session):
         == "Error for column ids: Cannot modify existing column with mutate()."
         " Use a different name for the new column."
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
[F] These tests were timing out under the old `DatasetQuery` code/methods so I migrated them to code that is currently being used. More context/discussion [here](https://iterativeai.slack.com/archives/C04A9RWEZBN/p1726114474925839).